Userland/Libraries/LibWeb/Painting/DisplayListPlayerSkia.h

```diff
@@ -23,7 +24,7 @@ class DisplayListPlayerSkia : public DisplayListPlayer {
 #endif
 
 #ifdef AK_OS_MACOS
-    DisplayListPlayerSkia(Gfx::SkiaBackendContext&, Core::MetalTexture&);
+    DisplayListPlayerSkia(RefPtr<Gfx::SkiaBackendContext>, NonnullRefPtr<Gfx::PaintingSurface>);
 #endif
```

Comment:
haven't finished scrolling through the commits, but it feels like we could remove the ifdefs here by using PaintingSurface consistently in the constructor.

Reply:
good catch! indeed I overlooked opportunity to remove ifdefs here.